akka-docs/src/test/java/jdocs/actor/typed/DispatcherDocTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package jdocs.actor.typed;
+
+import java.util.concurrent.Executor;
+
+import scala.concurrent.ExecutionContextExecutor;
+
+import akka.actor.typed.*;
+import akka.actor.typed.javadsl.*;
+
+@SuppressWarnings("unused")
+public class DispatcherDocTest {
+
+  private final ActorSystem<Void> system = null;
+  private final ActorContext<Void> context = null;
+
+  public void defineDispatcherInConfig() {
+    // #defining-dispatcher-in-config
```

Comment:
I can't see that these snippet anchors are used anywhere?

BTW, I think we should recommend the `fromConfig` ones over the somewhat magic path based config.